### DISCUSSION TITLE
Option to define system email messages as HTML in addition to plain text

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9189] [Ideas]
> Created by: brad (brad@pixelandtonic.com) on 2015/03/06 19:07:45 +0000
> Votes at time of UserVoice import: 5

-----------------------------------

Currently anything entered in those fields are treated as plain text and that are ran through a Markdown parser to generate the HTML body portion of the email.

However for things that include links (like user activation email, password reset emails, etc.) there are some email clients that will not auto-hyperlink any URLs they see making the client copy and paste the URL in order to use it.

If there were separate fields, you could customize both the plain text and HTML email bodies.

**Comment by angrybrad** [comment]
Related to #1107.

**Comment by brandonkelly** [comment]
Links can be handled with markdown syntax (`[link](http://foo.com)`), so that's not a great argument for this, but not 100% against the idea.

**Comment by RickKukiela** [comment]
@brandonkelly damn I had no idea you could do that link markdown thing... A helper tooltip with this info would have been great. 

Also there is a work around which involves pointing the HTML template to a twig file that does this:

```
{% extends 'mail/_layout' %}

{% block content %}

    {% if emailKey == 'account_activation' %}
        {% include 'mail/activation' %}
    {% endif %}
    {% if emailKey == 'forgot_password' %}
        {% include 'mail/forgot-password' %}
    {% endif %}
    {% if emailKey == 'test_email' %}
        {% include 'mail/test-email' %}
    {% endif %}
    {% if emailKey == 'verify_new_email' %}
        {% include 'mail/verify-new-email' %}
    {% endif %}

{% endblock %}
```
